src/cobra/test/test_core/test_gpr.py

```diff
@@ -48,8 +48,8 @@ def test_one_gene_gpr() -> None:
 
 # Gets an iterable of all combinations of genes except the empty list. Used to
 # evaluate AND gprs
-def powerset_ne(iterable):
-    """powerset_ne([1,2,3]) --> (1,) (2,) (3,) (1,2) (1,3) (2,3) (1,2,3)"""
+def powerset_ne(iterable: Iterable) -> Iterator:
```

Comment:
Is it of gene?

```suggestion
def powerset_ne(iterable: Iterable[Gene]) -> Iterator[Gene]:
```

Reply:
No. Currently this is an Iterable of strings, used on a set.
So I guess it could be Iterable[str], but I'm not sure that is very helpful. Do you want me to change it to that?

Reply:
It's true that it's just in the tests but I still think it's helpful, yes.

Reply:
Done